materials/study_02/interview-planning.md

```diff
@@ -26,4 +26,30 @@ Suggestion 2: DATE and TIME
 Suggestion 3: DATE and TIME
 
 Kind regards,
-Jan Volkel 
\ No newline at end of file
+Jan Voelkel 
+
+
+## Reminder 3 days before the interview
+
+Dear ___,
+
+This is a friendly reminder that the interview for the study on data fabrication in which you participate will take place in 3 days.
+
+Please make sure that you have fabricated the data by then.
```

Comment:
Add a link to the instructions so they can re-download if they need to. This is the link:
https://surfdrive.surf.nl/files/index.php/s/mMDkgCILKU8rhhD